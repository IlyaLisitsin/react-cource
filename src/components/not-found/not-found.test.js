import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from './not-found.component';

it('renders NotFound correctly', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
});