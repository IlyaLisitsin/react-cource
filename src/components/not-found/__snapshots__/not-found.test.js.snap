// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NotFound correctly 1`] = `
<img
  alt="not found"
  src="https://weeblytutorials.com/wp-content/uploads/2017/05/Weebly-404-Page-Not-Found-Error1.png"
/>
`;
